controllers/etcdadmconfig_controller.go

```diff
@@ -404,7 +404,7 @@ func (r *EtcdadmConfigReconciler) storeBootstrapData(ctx context.Context, config
 					Kind:       config.Kind,
 					Name:       config.Name,
 					UID:        config.UID,
-					Controller: pointer.Bool(true),
+					Controller: ptr.To[bool](true),
```

Comment:
nit: do you need the `[bool]`? the compiler should be able to infer that

Reply:
That is true!